expenses-manager-frontend/src/main/frontend/src/app/daily-stats/daily-stats.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="dashboard-container">
+
+  <div class="date-picker-container">
+    <button mat-icon-button (click)="previousDay()">
+      <mat-icon>chevron_left</mat-icon>
+    </button>
+
+    <button mat-button (click)="picker.open()">
+      {{ currentDate | date: 'EEEE - d MMMM y' }}
+    </button>
+
+    <button mat-icon-button (click)="nextDay()">
+      <mat-icon>chevron_right</mat-icon>
+    </button>
+
+    <input matInput [matDatepicker]="picker" (dateChange)="onDateChange($event)" [max]="maxDate" style="display: none;">
```

Comment:
Am vazut ca date picker-ul se afiseaza in partea de sus stanga a paginii, ar fi frumos sa se afiseze sub inputul de data, peste chart